README.md

```diff
@@ -26,30 +35,53 @@ Dependencies
+Accepts rank-two input tensors, each is fed-forward through a configured artificial neural network, which predicts values for each of its outputs. Training and testing data will be compared to their empirical labels and evaluated for loss as defined by the user. During training weights are updated through back-propogration, according to the optimizer selected. The default configuration is ready to take input from an MNISTImageLoader block, and will surpass 90% accuracy in about 2,000 training steps.
 
 Properties
 ----------
-- **enrich**: *enrich_field:* The attribute on the signal to store the results from this block. If this is empty, the results will be merged onto the incoming signal. This is the default operation. Having this field allows a block to 'save' the results of an operation to a single field on an incoming signal and notify the enriched signal.  *results field:* The attribute on the signal to store the results from this block. If this is empty, the results will be merged onto the incoming signal. This is the default operation. Having this field allows a block to save the results of an operation to a single field on an incoming signal and notify the enriched signal.
-- **layers**: Define one or more network layers with the specified number of neurons, a bias unit if selected, an activation function, and initial weight values. Each layer's input is the layer before it (or input data, in the case of the first layer).
-- **network_config**: Parameters of the artifical neural network. Dropout is only used if one or more dropout layers are configured in the network.
+- **enrich**: Signal Enrichment
+  - *exclude_existing*: If checked (true), the attributes of the incoming signal will be excluded from the outgoing signal. If unchecked (false), the attributes of the incoming signal will be included in the outgoing signal.
+  - *enrich_field*: (hidden) The attribute on the signal to store the results from this block. If this is empty, the results will be merged onto the incoming signal. This is the default operation. Having this field allows a block to 'save' the results of an operation to a single field on an incoming signal and notify the enriched signal.
+- **layers**: Define one or more network layers. Each layer's input is the layer before it (or input data, in the case of the first layer).
+  - *count*: Number of neurons in this layer
+  - *activation*: Activation function, use *bias_add* to use no activation function.
+  - *initial_weights*: Initialize newly created model weights with random or fixed values.
+  - *bias*: Add a bias unit to this layer's input.
+- **models**: Visualize models using [TensorBoard](https://www.tensorflow.org/get_started/summaries_and_tensorboard#launching_tensorboard). Save and Load training progress in [Checkpoint Files](https://www.tensorflow.org/versions/master/get_started/checkpoints).
```

Comment:
May be worth adding in that line required to run tensorboard locally `tensorboard --logdir=path/to/log-dir`.  Eliminates the possibility of confusion

Reply:
links to "launching tensorboard" page tho, which states exactly that and plausibly will be maintained for future updates

Reply:
Didn't think about future updates/changes.  Good call